src/lib/workspace/Subsetting/SubsetDownloadModal.tsx

```diff
@@ -706,21 +706,32 @@ export default function SubsetDownloadModal({
 
   // ~18px (round to 20px) per character for medium title size
   const maxStudyNameLength = Math.floor(modalHeaderWidth / 20);
-  const studyName =
-    studyRecord.displayName.length > maxStudyNameLength ? (
-      <span title={studyRecord.displayName}>
-        {safeHtml(
-          studyRecord.displayName.substring(0, maxStudyNameLength - 2) + '...'
-        )}
+  const regexAllTags = /<.*?>/g;
+  const studyNameNoTags = studyRecord.displayName.replaceAll(regexAllTags, '');
```

Comment:
I did a similar thing not long ago, and it was suggested that I reach instead for [this `stripHTML` utility function](https://github.com/VEuPathDB/WDKClient/blob/1d179fe1825dac38134273fa0e1d8de670e96f22/Client/src/Utils/DomUtils.ts#L285-L292) instead.

Reply:
The current solution can introduce false positives:
![image](https://user-images.githubusercontent.com/365139/181616285-e4df344a-4fdf-4f6f-aad4-73b717f79b73.png)